src/shared/components/Header/Navigation/component.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Navigation, ToggleMenu, SearchButton } from './styles';
+import { MenuListItemI } from '../interfaces';
+import SVGSearch from '../../../icons/Search';
+
+type NavigationProps = {
+  menuList: MenuListItemI[];
+  onToggle(type: string): void;
+};
+
+const Header: React.FC<NavigationProps> = (props: NavigationProps) => {
+  return (
+    <Navigation>
+      <ul>
+        <li>
+          <SearchButton
+            onClick={(): void => {
```

Comment:
non-blocking: could be one-liner. 

one-liners easier to read and save space

Reply:
eslint displays an error

Reply:
what error?

Reply:
this?

```tsx
<SearchButton onClick={(): void => props.onToggle('search')}>

```